content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,134 @@ weight: 22033
 
 # Introduction
 
-Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
+Private Locations facilitate running simulations in your dedicated cloud environment. 
+Ensure secure storage of sensitive simulation packages in your private cloud. 
 
-Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
+The Control Plane offers a private repository; enable it for confidential package management!
 
 # Private packages
 
-## Infrastructure
+To create private packages, utilize our Gatling Enterprise Cloud packages API and upload them through a 
+plugin configured with the Control Plane's private package repository and a Configurer public API Token.
+
+When initiating a Gatling run, the Control Plane generates a temporary signed link to allow the download of the private package from the load generators.
 
 {{< img src="private_packages_general_architecture.png" alt="Infrastructure schema" >}}
 
 {{< alert info >}}
 Private Packages are based on Private Locations. You can not use them with managed locations.
 {{< /alert >}}
 
-## Installation
+## Infrastructure
 
-First step: follow the control plane installation guide for private locations.
+Currently, private packages can be integrated with either AWS S3 or GCP Cloud Storage repositories.3
 
 {{< alert info >}}
-Before going further, ensure that your AWS S3 bucket is ready to hold your packages.
+Before going further, ensure that your repository is ready to hold your packages.
 {{< /alert >}}
 
-{{< alert warning >}}
-Control plane with private repository needs AWS permissions `s3:PutObject` and `s3:DeleteObject` on the bucket.
-{{< /alert >}}
+### Control Plane server
 
-Once it is done, add the private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}}) file:
+First thing first, control plane with private repository have a server to forward uploads on the repository using 
+Gatling Enterprise Cloud API Token with Configurer permission.
+
+The server is accessible on port 8080 by default when a repository is configured.
+Below, you'll find the server configuration along with its default settings for your reference.
 
 ```bash
 control-plane {
   repository {
-    # S3 Bucket configuration
-    bucket = "bucket-name"
-    path = "folder/to/upload" # Optional, default to root, do not set trailing slash
+    # Upload configuration (optional)
```

Comment:
What we've done lately: optional configurations are documented as comments so that users do not leave them in their own conf at copy paste if they don't need it.

Reply:
Here, it'll result in a fully commented block. I specified in bold that this is optional.
That way, they only copy repository bucket configuration